packages/s2-core/src/common/interface/basic.ts

```diff
@@ -151,6 +151,10 @@ export interface Total {
   label?: string;
   // sub label's display name, default = '小计'
   subLabel?: string;
+  /** 总计分组维度 */
+  totalsDimensionsGroup?: string[];
```

Comment:
个人觉得反过来更表意, 和 `subTotalsDimensions` 类似, 接收的都是 `string[]`, 这里表示的是 指定多个维度, 然后我们来帮你分组, `xxDiemensions` 更合理一点, 也和其他命名风格保持一致

```suggestion
  totalsGroupDimensions?: string[];
  subTotalsGroupDimensions?: string[];
```

Reply:
修改为subTotalsDimensions